test/unit_tests/braket/tasks/test_gate_model_quantum_task_result.py

```diff
@@ -234,6 +331,29 @@ def test_openqasm_shots_calculate_result_types(openqasm_result_obj_shots):
 ]
 
 
+def test_get_compiled_circuit_rigetti(result_rigetti, quil_program):
+    """Test get_compiled_circuit method."""
+    assert result_rigetti.get_compiled_circuit() == quil_program
+
+
+def test_get_compiled_circuit_oqc(result_oqc, qasm2_program):
```

Comment:
nit: several of these tests/fixtures can be parametrized with pytest instead of duplicated

Reply:
It looks like I should be able to do something like this -- but afaiu parameter inputs can't be fixtures. Do you know how to do it, beyond accepting the fixture name as a string and looking up the function from it? If you have other more specific recommendations please let me know.

```
@pytest.mark.parametrize("result,program", [(result_rigetti, quil_program), (result_oqc, qasm2_program)])
def test_get_compiled_circuit(result, program):
    """Test get_compiled_circuit method."""
    assert result.get_compiled_circuit() == program
```

Reply:
Updated the result fixtures to remove duplicated code